pinax/likes/urls.py

```diff
@@ -2,6 +2,8 @@
 
 from . import views
 
+app_name = "likes"
```

Comment:
Use `"pinax_likes"` to provide an obviously unique namespace and match our documentation.